samples/cellular/modem_shell/README.rst

```diff
@@ -42,7 +42,7 @@ Many of the changes are applied when going to online mode for the next time.
 You can store some link subcommand parameters into settings, which are persistent between sessions.
 
 3GPP Release 14 features are enabled by default, which means they are set when going into normal mode and when booting up.
-For the list of supported features, refer to `3GPP Release 14 features AT command`_.
+For the list of supported features, refer to the `3GPP Release 14 features AT command`_ section in the nRF9160 AT Commands Reference Guide or the same section in the `nRF91x1 AT Commands Reference Guide`_ depending on the SiP you are using.
 To disable these features in normal mode,  use the ``link funmode --normal_no_rel14`` command.
 During autoconnect in bootup, use the ``link nmodeauto --enable_no_rel14`` command.
```

Comment:
This section about disabling/enabling Release 14 features is only valid for nRF9160. With nRF91x1, Release 14 features are always enabled and can not be disabled, so this needs to be rephrased a bit.

Reply:
Updated. please check.